src/http.js

```diff
@@ -0,0 +1,54 @@
+import { expandReferences } from "../";
+import axios from "axios";
+
+/**
+ * Make a GET request
+ * @public
+ * @example
+ *  get("/myendpoint", {
+ *      params: {foo: "bar", a: 1},
+ *      headers: {"content-type": "application/json"}
+ * })
+ * @function
+ * @requestParams {object} request parameters passed to Axios
+ * @returns {Operation}
+ */
+export function get(requestParams) {
+  return (state) => {
+    const params = expandReferences(requestParams)(state);
+
+    return axios({ method: "get", ...params });
```

Comment:
The current implementation of `language-http` calls `composeNextState` to shift old stuff into the `references` array, pop the response into the `data` key, and provide a pretty specific shape to the `state` that comes out of the bottom of one of these operations. Here you're not doing that, and I sense it's because we want these to be low-level... i.e., if you're using these guys in an expression, you want full control yourself. Is that right?